gitlab/resource_gitlab_project_freeze_period.go

```diff
@@ -0,0 +1,148 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabProjectFreezePeriod() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabProjectFreezePeriodCreate,
+		Read:   resourceGitlabProjectFreezePeriodRead,
+		Update: resourceGitlabProjectFreezePeriodUpdate,
+		Delete: resourceGitlabProjectFreezePeriodDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
`project_id` should have `ForceNew: true` since `project_id` is not a valid update option.